posthog/queries/trends/sql.py

```diff
@@ -296,6 +303,23 @@
 ) WHERE 11111 = 11111 {parsed_date_from} {parsed_date_to}
 """
 
+BREAKDOWN_ACTIVE_USER_AGGREGATE_SQL = """
+SELECT
+    {aggregate_operation} AS total, breakdown_value
+FROM (
+    SELECT toStartOfDay(toTimeZone(toDateTime(timestamp, 'UTC'), %(timezone)s)) as timestamp, {person_id_alias}.person_id AS person_id, {breakdown_value} as breakdown_value
```

Comment:
Could we format this in a cleaner way to make this easier to read?

- Make SQL in-builts capitalized
- One thing per line
```suggestion
    SELECT 
      toStartOfDay(toTimeZone(toDateTime(timestamp, 'UTC'), %(timezone)s)) AS timestamp,
      {person_id_alias}.person_id AS person_id, 
      {breakdown_value} AS breakdown_value
```